sale_stock_on_hand_popup/wizards/product_quant_wizard.py

```diff
@@ -0,0 +1,22 @@
+from odoo import api, fields, models
+
+
+class ProductQuantWizard(models.TransientModel):
+    _name = "product.quant.wizard"
+    _description = "Product Quant Wizard"
+
+    product_id = fields.Many2one("product.product")
+    stock_quant_ids = fields.Many2many(
+        "stock.quant", compute="_compute_stock_quant_ids"
+    )
+
+    @api.depends("product_id")
+    def _compute_stock_quant_ids(self):
+        for rec in self:
+            rec.stock_quant_ids = rec.product_id.stock_quant_ids.filtered(
+                lambda x: x.location_id.usage in ["internal", "transit"]
+                if self.env.user.has_group(
```

Comment:
I'm not sure about this. Couldn't this leads to operational misunderstandings as stock user not in that group won't see the product quantities even if they are there.

Reply:
They are not technically there, we are excluding quants that are in "transit", if the user doesn't have the group it's still gonna be seeing all of the quants that are in a location of type "internal".

It depends on the company if they want to keep in consideration also quants that are in transit or not.

Reply:
Wonder if "it depends on the company" should also translate to a company-related setting rather than a setting for the whole installation...